src/client/src/rt-components/platform/adapters/openfin/openFin.ts

```diff
@@ -108,11 +118,35 @@ export default class OpenFin extends BasePlatformAdapter {
   }
 
   notification = {
-    notify: (message: object) =>
-      new fin.desktop.Notification({
-        url: '/notification',
-        message,
-        timeout: 8000,
-      } as any),
+    notify: (message: object) => {
+      this.createNotification(
+        Object.assign({
+          body: this.getNotificationBody(message),
+          title: this.getNotificationTitle(message),
+          icon: `${location.protocol}//${location.host}/static/media/icon.ico`,
+          customData: message,
+          buttons: [{ title: 'Highlight trade in blotter' }],
+        }),
+      )
+        .then(successVal => {
+          console.info('Notification success', successVal)
+        })
+        .catch(err => {
+          console.error('Notification error', err)
+        })
+    },
+  }
+
+  getNotificationTitle({ tradeNotification }: NotificationMessage) {
+    const status = tradeNotification.status === 'done' ? 'Accepted' : 'Rejected'
+    return `Trade ${status}: ${tradeNotification.direction} ${tradeNotification.dealtCurrency} ${
```

Comment:
I'm not sure this is the best place for the Notification content to be generated, what if we want to support different types of notifications? Is there a way we can move this out of the Platform adapter?

Reply:
We already have the 'NotificationMessage' which is the notification across platform. We need this code here because it is specific to OpenFin. Each platform has their own constraints/format... When we have other types of notification to handle, then we can maybe refactor the code...